community/kernel/src/test/java/org/neo4j/kernel/impl/store/IdGeneratorRebuildFailureEmulationTest.java

```diff
@@ -75,7 +73,6 @@ protected void emulateFailureOnRebuildOf( NeoStores neoStores )
         }
     }
 
-    @BreakpointTrigger
     private void performTest() throws Exception
```

Comment:
Do these tests still work like before? I'm not seeing what the break points were actually doing in this case.

A small additional cleanup could be to pass the id file name as a parameter, instead of computing it from the stack trace.